core/object/object.h

```diff
@@ -676,6 +676,9 @@ class Object {
 
 	Object(bool p_reference);
 
+	template <bool Reversed>
+	_FORCE_INLINE_ void _notification_fast(int p_notification);
```

Comment:
I'm not sure about `_FORCE_INLINE_`. It's hard to judge. The function is not as small as typical inliners. I'd suggest removing the specifier and let the compiler apply its judgement.